src/gui/render/sky.hpp

```diff
@@ -46,10 +46,9 @@ namespace render {
  */
 class SkyRenderer {
  private:
-    GLuint programID_; // ID of Program
-    // ID of world space to camera space transform matrix
-    GLuint matrix_view_projection_ID_;
-    GLuint pixel_matrix_ID_;            // ID of view space to pixel space matrix
+    GLuint programID_;                  // ID of Program
+    GLuint matrix_view_projection_ID_;  // ID of world to camera space transform matrix
+    GLuint pixel_matrix_ID_;            // ID of view to pixel space matrix
```

Comment:
```suggestion
    GLuint program_id_;                  
    GLuint matrix_view_projection_id_;  // ID of world to camera space transform matrix
    GLuint pixel_matrix_id_;            // ID of view to pixel space matrix
```

Reply:
Did you do this?

Reply:
https://github.com/AlemSnyder/Fun-Game/pull/47#discussion_r1271471753
This will be refactored in a pr dedicated to shaders.